pywbem/_listener.py

```diff
@@ -1572,30 +1568,30 @@ def _handle_indication(self, indication, host):
           queue.Full: Indication queue is full
         """
         if self._ind_queue is None:
-            self.logger.warning(
-                "_handle_indication: Indication queue not set up - "
-                "ignoring indication")
+            self.logger.error(
+                "Indication queue not set up - ignoring indication")
             return
 
         try:
             # Non-blocking put is required to raise queue.Full exception if full
-            self._ind_queue.put((indication, host), block=False)
+            queue_item = (indication, host, msgid)
+            self._ind_queue.put(queue_item, block=False)
```

Comment:
good place for one comment about this piece of logic on next line since the logic is sort of inverted.  I.e. Put log indicating that queue is no longer full.